test/test_miscellaneous_helper_functions.py

```diff
@@ -0,0 +1,24 @@
+import appdaemon.plugins.hass.hassapi as hass
+
+from appdaemontestframework import automation_fixture
+
+COVER = 'cover.some_cover'
+
+class WithMiscellaneousHelperFunctions(hass.Hass):
+    def initialize(self):
+        pass
+
+    def get_entity_exists(self, entity):
+        return self.entity_exists(entity)
+
+@automation_fixture(WithMiscellaneousHelperFunctions)
+def with_miscellaneous_helper_functions():
+    pass
+
+
+def test_entity_exitsts_true(given_that, with_miscellaneous_helper_functions, hass_functions):
```

Comment:
Small typo in the test name: 'exitsts' -> 'exists'

Reply:
Done.